### PR TITLE
Overview of most important changes

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds new terms of use with versions

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5466/commits/c72842afeae1e14a13d5932ccf5c6943d9bda002 is at https://pr5466.build.csb.dev/s/new.

File: packages/app/src/app/overmind/internalActions.ts [hunk 1]
```diff
@@ -542,11 +542,18 @@ export const showPrivacyPolicyNotification: Action = ({ effects, state }) => {
     effects.analytics.track('Saw Privacy Policy Notification');
     effects.notificationToast.add({
       message:
-        'Hello, our privacy policy has been updated recently. What’s new? CodeSandbox emails. Please read and reach out.',
+        'Hello, we are changing our Terms of Use effective Mar 14, 2021 12:00 UTC. Please read them, or see commit message for a brief sum up.',
       title: 'Updated Privacy',
       status: NotificationStatus.NOTICE,
       sticky: true,
       actions: {
+        secondary: {
```
Comment (SaraVieira) [inline packages/app/src/app/overmind/internalActions.ts:550]: As soon as merge come back to get link and paste here

File: packages/app/src/app/overmind/internalActions.ts [hunk 2]
```diff
@@ -532,6 +532,7 @@ export const identifyCurrentUser: AsyncAction = async ({ state, effects }) => {
   }
 };
 
+// add ACCEPTED_TERMS_CODESANDBOX_v1.1
 const seenTermsKey = 'ACCEPTED_TERMS_CODESANDBOX';
```
Comment (SaraVieira) [inline packages/app/src/app/overmind/internalActions.ts:536]: Change after merge

File: packages/homepage/src/components/Toast/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { AnimatePresence } from 'framer-motion';
 
 import { ToastContainer } from './elements';
 
+// add ACCEPTED_TERMS_CODESANDBOX_v1.1
```
Comment (SaraVieira) [inline packages/homepage/src/components/Toast/index.js:8]: Change after merge

**Review by oskarabc — COMMENTED** [review]
Proposing versioning: https://codesandboxapp.slack.com/archives/CHM6FA0TA/p1613123939018400

**Pull request merged**